### PR TITLE
Security issues reported by Snyk

### PR DESCRIPTION
**Pull request by imakowski**
Security vulnerabilities reported by Snyk

### CODE REVIEW

**Comment by imakowski** [comment]
@raullenchai @dustinxie @huangzhiran Please review and merge

**Comment by imakowski** [comment]
@Liuhaai please merge

**Comment by imakowski** [comment]
@dustinxie @Liuhaai @raullenchai @CoderZhi Can you review and merge?

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/ethereum/go-ethereum v1.10.4
+	github.com/ethereum/go-ethereum v1.10.23
```
Comment (dustinxie) [inline go.mod:7]: same as last PR, why is this necessary?
can you provide a link to the security issue related to v1.10.4?

Comment (dustinxie) [reply go.mod:7]: also, pls run `go mod` each time (and before your submit PR)

Comment (imakowski) [reply go.mod:7]: Support for 1.18.5 is added in this version? Why we cannot upgrade Ethereum to this version?

Comment (imakowski) [reply go.mod:7]: > also, pls run `go mod` each time (and before your submit PR)

Ok

**Comment by imakowski** [comment]
Any update?

File: go.mod (hunk 1 above)
Comment (dustinxie) [reply go.mod:7]: can you provide a link to the security issue related to v1.10.4， which was solved in v1.10.23?

**Comment by dustinxie** [comment]
close this since no satisfactory explanation is given
